harness/determined/launch/wrap_rank.py

```diff
@@ -61,13 +62,16 @@ def main() -> int:
         rank = int(args.rank)
     else:
         # Rank is provided as the name of an environment variable.
-        if args.rank not in os.environ:
+        for rank in args.ranks.split(","):
```

Comment:
You need rank=None at 59:

Reply:
I think this time there is no way to get through without rank being defined like before. but I'll add anyway because I don't like python

Reply:
fixed my terrible code though, too